apikey-service/src/main/java/eu/europeana/apikey/controller/ApikeyController.java

```diff
@@ -143,6 +244,31 @@ public ApikeyController(ApikeyRepo apikeyRepo) {
         }
     }
 
+    /**
+     * Get value from the Authorization header of the given request based on the supplied pattern.
+     *
+     * @param httpServletRequest request with the header
+     * @param valuePattern pattern of the Authorization header to retrieve the value
+     * @return value of the Authorization header
+     */
+    private String getAuthorizationHeader(HttpServletRequest httpServletRequest, String valuePattern) {
+        String authorization = httpServletRequest.getHeader("Authorization");
```

Comment:
Very minor issue, but `getHeader("Authorization")` can be replaced with `getHeader(HttpHeaders.AUTHORIZATION)`